src/commands/chain.rs

```diff
@@ -36,7 +37,16 @@ pub(crate) enum Command {
         limit: u8,
     },
     #[clap(about = "Retrieves details for a given L1 batch.")]
-    L1BatchDetails { batch: u32 },
+    L1BatchDetails {
+        #[clap(short = 'n', num_args = 1..)]
+        batches: Option<Vec<L1BatchNumber>>,
```

Comment:
Let's try this as an `Option<Vec<T>>` can be treated as an empty `Vec<T>`, if `--batches` is not passed, then the default will be an empty `Vec`.
```suggestion
        #[clap(short = 'n', num_args = 1.., default_value_t = Vec::default)]
        batches: Option<Vec<L1BatchNumber>>,
```

Reply:
Then the logic below will be simplified.